analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -44,8 +44,7 @@ typealias ResolutionResult = MutableMap<File, AnalyzerResult>
  */
 abstract class PackageManager {
     companion object {
-        @JvmStatic
-        val HTTP_CACHE_PATH = "analyzer/cache/http"
+        const val HTTP_CACHE_PATH = "analyzer/cache/http"
```

Comment:
Actually, what I meant by [use the constant](https://github.com/heremaps/oss-review-toolkit/pull/168#issuecomment-353305302) was to use `${Main.TOOL_NAME}` also here, but making the `val` a `const` is also a good change :-) Mind doing both?

Reply:
Done.